### PR TITLE
Feature/cli load keys

### PR DESCRIPTION
**Pull request by sshaman1101**
This PR adds the `login` method for CLI. The method allows creating or opening an Etherum keystore.

TBD:
- [x] extract key loading logic implemented in loginCmd command;
- [x] Add a "middleware" (using _cobra.Command.PreRunE_) to load the storage on each action that must be signed;

### CODE REVIEW

**Comment by sshaman1101** [comment]
@noxiouz @3Hren PTAL

File: cmd/cli/commands/common.go [hunk 1]
```diff
@@ -151,3 +156,40 @@ func isSimpleFormat() bool {
 
 	return true
 }
+
+// printer interface describe anything that can print
+// something somehow on a something.
```
Comment (3Hren) [inline cmd/cli/commands/common.go]: Such abstraction
So java
Much whatever
Wow.

File: util/util.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/ethereum/go-ethereum/crypto"
 	"gopkg.in/yaml.v2"
+	"os"
```
Comment (3Hren) [inline util/util.go]: Sort.

File: util/util.go [hunk 3]
```diff
@@ -116,3 +117,11 @@ func LoadYamlFile(from string, to interface{}) error {
 
 	return nil
 }
+
+// DirectoryExists returns true if given directory exists
```
Comment (3Hren) [inline util/util.go]: The given.

**Pull request merged**